README.md

```diff
@@ -1,4 +1,4 @@
-# Benchmarks
+﻿# Benchmarks
```

Comment:
It appears your text editor inserted the UTF-8 Byte Order Mark (BOM), while the original file didn't have a BOM.  Can you please remove the BOM?